components/encryption/src/master_key/kms.rs

```diff
@@ -68,10 +98,13 @@ impl KmsBackend {
         let mut opt_state = self.state.lock().unwrap();
         if opt_state.is_none() {
             let mut runtime = self.runtime.lock().unwrap();
-            let data_key = self.kms_provider.generate_data_key(&mut runtime)?;
+            let timeout_duration = Duration::from_secs(10);
```

Comment:
make it a constant?

Reply:
moved it into the struct. It should be configurable in the future.